users/ignoring.rst

```diff
@@ -17,10 +17,10 @@ Description
 
 If some files should not be synchronized to (or from) other devices, a file called
 ``.stignore`` can be created containing file patterns to ignore. The
-``.stignore`` file must be placed in the root of the folder. The
+``.stignore`` file must be placed in the root of the synced folder (see :ref:`Folder concepts <getting-started>`). The
```

Comment:
Following that link to the getting started guide I don't find what "Folder concepts" refers to.

Reply:
Am I missing something - the only bit about folders there I find is a description of the UI:

> On the left is the list of “folders”, or directories to synchronize. You can see the Default Folder was created for you, and it’s currently marked “Unshared” since it’s not yet shared with any other device. On the right is the list of devices. Currently there is only one device: the computer you are running this on.

I don't see how this is about folder concepts or helpful in the context of understanding putting a .stignore into the synced folder root.

Reply:
Yup, removing sounds good. And a concepts page would be nice too, respectively maybe add it to (Understanding Synchronisation](https://docs.syncthing.net/users/syncing.html), that's close to concepts I'd say.